cfme/rest/gen_data.py

```diff
@@ -42,10 +41,11 @@ def service_catalogs(request, rest_api, num=5):
     return _creating_skeleton(request, rest_api, 'service_catalogs', scls_data, col_action='add')
 
 
-def service_catalog_obj(request, rest_api):
+def service_catalog_obj(appliance, request, rest_api):
```

Comment:
**REQUIRED:** As we are changing this anyway, can we drop the rest_api argument. when we invoke ```service_catalogs```, we can just pass, ```appliance.rest_api``` instead.